src/Entity/OgMembershipType.php

```diff
@@ -42,4 +58,35 @@ public function id() {
     return $this->type;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function save() {
+    $status = parent::save();
+
+    if ($status === SAVED_NEW) {
+      FieldConfig::create([
+        'field_name' => 'og_membership_request',
+        'entity_type' => 'og_membership',
+        'bundle' => $this->id(),
+        'label' => 'Request Membership',
+        'description' => 'Explain the motivation for your request to join this group.',
+        'translatable' => TRUE,
+        'settings' => [],
+      ])->save();
+    }
+
+    return $status;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function delete() {
+    if ($this->id() === 'default') {
```

Comment:
@jludwig 
 @`\Drupal\og\OgMembershipInterface::TYPE_DEFAULT` exists, would be nicer to have that.